index.html

```diff
@@ -9,6 +9,32 @@
   <body class="page">
     <div class="page__container">
       <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+      <section class="contacts">
+        <h2 class="contacts__title">Остались вопросы?Свяжитесь с нами!</h2>
+        <ul class="contacts__list">
+          <li class="contacts__item">
+            <h3 class="contacts__heading" >Звоните:</h3>
+            <p class="contacts__access">+7(812) 555-66-66</p>
+          </li>
+          <li class="contacts__item contacts__item_type_adress">
```

Comment:
Ooops, a little typo here. The correct spelling of the word is **address**.

Reply:
@SergeyAlexandrowich HTML only? 🙈

Reply:
@naumch1k   Ooooops, I have forgotten. However I've already fixed it